app/code/community/Elgentos/CodebaseExceptions/Helper/Data.php

```diff
@@ -41,4 +41,35 @@ public function insertException($reportData) {
 
         $this->client->notifyOnError($reportData[0],$backtraces);
     }
+
+    /**
+     * @param string $message
+     */
+    public function sendToAirbrake($message) {
+        $backtraceArray = explode("\n", $message);
+        if (count($message) < 1) {
+            return;
+        }
+        $errorMessage = '';
+        //reformatting exception
+        if (isset($backtraceArray[0])) {
+            $errorMessage .= $backtraceArray[0];
+            unset($backtraceArray[0]);
+        }
+        if (empty($errorMessage) && isset($backtraceArray[1])) {
```

Comment:
when first array item is empty => $message had a line break at the beginning. This is most common case actually.
We want to use first line as a error message, and put the rest into backtrace.